Automattic/wp-calypso
# Adding Paid Newsletter card to the Earn page.

This PR adds a Paid Newsletter card to the Earn page. For users who have access to the feature, the card CTA points people to the support page that describes how to use it. For users who don't have access, the CTA prompts them to unlock the feature and takes them to the Plans page.

Before:
<img width="1055" alt="Screen Shot 2020-06-05 at 9 39 58 AM" src="https://user-images.githubusercontent.com/35781181/83883529-95ef6c00-a711-11ea-9a4f-5bde78404be0.png">

After (Free plan):
<img width="1057" alt="Screen Shot 2020-06-05 at 9 48 20 AM" src="https://user-images.githubusercontent.com/35781181/83883607-b6b7c180-a711-11ea-80e9-19ddc01dc143.png">

After (paid plan):
<img width="531" alt="Screen Shot 2020-06-05 at 9 49 51 AM" src="https://user-images.githubusercontent.com/35781181/83883767-ee266e00-a711-11ea-8b90-8ac8ae824f4a.png">

#### Testing instructions
* Checkout branch (git checkout -b update/earn-copy-tweaks origin/update/earn-copy-tweaks)
* Start Calypso locally
* Visit /earn with a free plan and a paid plan.
* Free plan: confirm that the card displays properly, that the CTA is "Unlock this feature," and that clicking takes you to the Plans page.
* Paid plan (any plan): confirm that the card displays properly, that the CTA is "Learn how to get started," and that click takes you to the Paid Newsletter support page.
